mypy/message_registry.py

```diff
@@ -330,3 +330,6 @@ def with_additional_msg(self, info: str) -> ErrorMessage:
 NARROWED_TYPE_NOT_SUBTYPE: Final = ErrorMessage(
     "Narrowed type {} is not a subtype of input type {}", codes.NARROWED_TYPE_NOT_SUBTYPE
 )
+TYPE_VAR_TOO_FEW_CONSTRAINED_TYPES: Final = ErrorMessage(
```

Comment:
Can this message be reused in `semanal.py` where a `TypeVar cannot have only a single constraint` message is used inline? This should help with discoverability of two `TypeVar` syntax variants constraints, because they share some common argument validation logic.

Reply:
I agree, if there are not too many tests to update, we should reuse the same message for both old and new style variables.

Reply:
I made the error messages consistent.